workflows/namespaced-scope-chaos/workflow.yaml

```diff
@@ -0,0 +1,173 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: argowf-chaos-hello-world-pod-delete-
+spec:
+  entrypoint: argowf-chaos
+  serviceAccountName: argo-chaos
+  arguments:
+    parameters:
+      - name: adminModeNamespace
+        valueFrom:
+          configMapKeyRef:
```

Comment:
Where is the reference for this configmap

Reply:
This workflow can be triggered only from litmus-portal and is specifically designed for the namespaced mode of operation. The config map we are creating as a part of the portal installation. @ksatchit